src/components/ui/register/steps/StepPassword.tsx

```
@@ -0,0 +1,21 @@
+import * as S from '@/containers/register/style';
+import { RegisterFormData } from '@/types/register';
+import { memo } from 'react';
+
+interface Props { form: RegisterFormData; onChange: (k: keyof RegisterFormData, v: any) => void; }
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)